lib/gitignore.js

```diff
@@ -0,0 +1,21 @@
+const Dotignore = require('@bcoe/dotignore')
```

Comment:
For Node 4 support, this file needs `'use strict'` at the top. This fixes the failing tests for me locally.

Reply:
@nexdrew I guess I'll need to spin up a Windows VM ... grumble.

Reply:
@nexdrew smart 👍